main.tf

```diff
@@ -8,7 +8,7 @@ locals {
 }
 
 resource "aws_route53_zone" "main" {
-  count = var.create_route53_zone ? 1 : 0
+  count = var.create_vpc && var.create_route53_zone ? 1 : 0
```

Comment:
Why you want to control route35 zone creation with `create_vpc` variable?

Reply:
Brought back with commit https://github.com/lean-delivery/tf-module-aws-core/pull/11/commits/02bfa3563c39c60528ac475e8610738609ba8019